### PR TITLE
Allow to set title from msiframe

### PR DESCRIPTION
**Pull request by jusabatier**
This allow to set layer's title to use in TOC from msiframe GET request.

I also added some documentation on how to use it in the doc.

@tdipisa @landryb Can someone with write permission merge this update ?

### CODE REVIEW

File: docs/source/msiframe/index.rst [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+==============================================================
+
+Simples iFrames for MapStore2
+
+What is msIframe ?
+------------------
+
+It allow to simplify the use of iframes on external website, an example
+is communication purposes. With it, you can use pretty much the same
+parameters as for sViewer, what make the map iframe’s URL generation
```
Comment (landryb) [inline docs/source/msiframe/index.rst:11]: s/what/which/ ?

File: docs/source/msiframe/index.rst [hunk 2]
```diff
@@ -0,0 +1,125 @@
+msIframe - A mapper for simple GET URLs to configure MapStore2
+==============================================================
+
+Simples iFrames for MapStore2
+
+What is msIframe ?
+------------------
+
+It allow to simplify the use of iframes on external website, an example
```
Comment (landryb) [inline docs/source/msiframe/index.rst:9]: allows

File: docs/source/msiframe/index.rst [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+   http://sdi.georchestra.org/mapstore/msIframe/?lb=1
+
+layers {string}
+---------------
+
+Comma-separated list of geOrchestra layer names with namespace
+extension. This is the fastest way to set up a map because we use
+context’s defined catalogs. It assumes :
+
+-  layers are delivered by one of the context’s catalogs
```
Comment (landryb) [inline docs/source/msiframe/index.rst:52]: provided

File: docs/source/msiframe/index.rst [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+To specify a vendor CQL_FILTER for a layer, add “\***cql_filter” to the
+layer name if no alternative catalog or style is specified, otherwise
+use layer*catalog*style*cql_filter examples :
+
+::
+
+   http://sdi.georchestra.org/mapstore/msIframe/?layers=geor:sdi*localgs*geor_sdi*title%3DPIGMA
+   http://sdi.georchestra.org/mapstore/msIframe/?layers=geor:sdi***title%3DPIGMA
+
+**Warming :** CQL_FILTER have to be URL encoded
```
Comment (landryb) [inline docs/source/msiframe/index.rst:93]: warNing ;)

**Comment by jusabatier** [comment]
It's corrected

**Pull request merged**